src/runtime/web/index.js

```diff
@@ -1,6 +1,35 @@
-import { createSimpleContext } from './api';
-import { View } from '../../bindings/html/view';
+import { prioritizeLocales } from '../../intl/index';
+import { HTMLLocalization } from '../../bindings/html';
+import { documentReady, getResourceLinks, getMeta } from './util';
```

Comment:
So with this patch, you can have `runtime/gecko` which defines its own `getResourceLinks` (querying for `<messagebundle>`s) and doesn't have `getMeta` at all.